programs/bpf/c/src/noop.c

```diff
@@ -2,26 +2,20 @@
  * @brief Example C-based BPF program that prints out the parameters
  * passed to it
  */
-
 #include <solana_sdk.h>
 
-/**
- * Number of SolKeyedAccounts expected. The program should bail if an
- * unexpected number of accounts are passed to the program's entrypoint
- */
-#define NUM_KA 1
```

Comment:
Keeping this comment in some form to highlight that the program writer needs to know how many accounts to expect would be helpful.

Reply:
Done.  I also moved static_assert() into solana_sdk.h for >=C11 so now noop.c and noop.cc are effectively identical.